### DISCUSSION TITLE
Question about IGS repro3 products

### DISCUSSION THREAD

**Discussion by RoyanLiSu** [groops-devs/groops#150] [Q&A]
Dear Developers,
I need IGS repro3 products for one year but these products are provided from 1994 until 2020 as a unit file for download and the volume is so big(e.g. TUG0R03FIN_1994-2020_01D_30S_ATT.OBX.zip is about 46 GB). is there any way to download the data separately based on the year?
regards,
Royan

**Comment by RoyanLiSu** [comment]
I can not download data from this server, unfortunately, the speed of the internet is low and the data transfer is constantly interrupted and the server does not support "resume" it is only possible to download data from the beginning. how can I fix this problem?

**Reply by ParisaShafiei** [reply]
Hi @RoyanLiSu, 

You can find all the TUG products in IGS webpage based on GPS week number and download them separately: [https://cddis.nasa.gov/archive/gnss/products/repro3/]

With kind regards, 
Parisa

**Comment by tmayerguerr** [comment]
Dear Royan,
Maybe you can look at [https://igs.org/products/#orbits_clocks](https://igs.org/products/#orbits_clocks)
and there at CDDIS: [https://cddis.nasa.gov/archive/gnss/products/wwww/repro3](https://cddis.nasa.gov/archive/gnss/products/wwww/repro3)] (after registration)
regards
Torsten
